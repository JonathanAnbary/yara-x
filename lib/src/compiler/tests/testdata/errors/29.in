rule test {
  condition:
    test_proto2.foo
}