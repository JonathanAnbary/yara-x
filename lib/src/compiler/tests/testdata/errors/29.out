error: unknown field or method `foo`
   ╭─[line:3:17]
   │
 3 │     test_proto2.foo
   │                 ─┬─  
   │                  ╰─── this field or method doesn't exist
───╯
